build.gradle.kts

```diff
@@ -29,7 +29,7 @@ dependencies {
   runtimeOnly("org.postgresql:postgresql")
   implementation("com.fasterxml.uuid:java-uuid-generator:5.1.0")
   implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.8.8")
-  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.19.0")
+  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.18.3")
```

Comment:
Reverted as caused test failures, not sure how the previous PR was merged in as the tests failed